src/dotnet-svcutil/lib/tests/src/E2ETests.cs

```diff
@@ -204,12 +201,12 @@ public void NamespaceParam(string testCaseName, string options, bool expectSucce
 
         [Trait("Category", "BVT")]
         [Theory]
-        [InlineData("TypeReuse60", "net6.0")]
```

Comment:
rather than use a generic format, I do prefer to add net7.0 and net 8.0 here to make it explicitly clear.  However, I do see your point that we don't need to update this in the future.  Similar to another comment before, does this mean we also allow other earlier .net versions?  We do need to be careful here.

Reply:
The format is made generic because the .net version is actually not used at all in the test, in the case context it always creates a test.csproj targets to the latest .net version (currently it's net8.0 as defined in global.json) to test type-reuse feature.